.buildkite/scripts/steps/code_coverage/jest_parallel.sh

```diff
@@ -1,38 +1,69 @@
 #!/bin/bash
 
-set -uo pipefail
+set -euo pipefail
 
-JOB=${BUILDKITE_PARALLEL_JOB:-0}
-JOB_COUNT=${BUILDKITE_PARALLEL_JOB_COUNT:-1}
+export JOB=$BUILDKITE_PARALLEL_JOB
 
 # a jest failure will result in the script returning an exit code of 10
-
-i=0
 exitCode=0
+results=()
+
+if [[ "$1" == 'jest.config.js' ]]; then
+  TEST_TYPE="unit"
+else
+  TEST_TYPE="integration"
+fi
+
+export TEST_TYPE
+echo "--- downloading jest test run order"
+buildkite-agent artifact download jest_run_order.json .
+#configs=$(jq -r 'getpath([env.TEST_TYPE]) | .groups[env.JOB | tonumber].names | first' jest_run_order.json)
+configs=$(jq -r 'getpath([env.TEST_TYPE]) | .groups[env.JOB | tonumber].names | .[]' jest_run_order.json)
```

Comment:
Same here, is it smth we would like to keep as alternative?

Reply:
No, I'll drop it.